base/install-functions.sh

```diff
@@ -36,14 +36,36 @@ install_module() {
     cd -
 }
 
+install_iocs() {
+    iocs_name=$1
+    dependency_name=$2
+    release_modules="$3"
+
+    cd $iocs_name
+    get_module_path "$release_modules" > configure/RELEASE
+
+    make -j${JOBS} install
+
+    cd -
+}
+
 # Install module from GitHub tagged versions or URL
-install_github_module() {
+install_github() {
+    if [ "$1" == "-i" ]
+        then iocs=1; shift
+        else iocs=0
```

Comment:
We can use `true` and `false` here instead. BTW, this should make it easier to write the `if` statement later:

```shell
is_ioc = true

if $is_ioc; then
  # do something...
fi
```